pkg/cloudprovider/provider/anexia/instance.go

```diff
@@ -45,9 +45,8 @@ func (ai *anexiaInstance) ID() string {
 	return ai.info.Identifier
 }
 
-// TODO(xmudrii): Implement this.
 func (ai *anexiaInstance) ProviderID() string {
-	return ""
+	return ai.info.Identifier
```

Comment:
hm, `ID()` above checks if `ai.info` is nil .. not sure if this is always called late enough in lifecycle to not be a problem

Reply:
I guess we could also have that check here to be on the safe side.

Reply:
changed `ProviderID()` to just call `ID()`